tests/suite/test_smoke.py

```diff
@@ -51,9 +51,15 @@ def fin():
 
 @pytest.mark.ingresses
 @pytest.mark.smoke
+@pytest.mark.parametrize('ingress_controller',
+                         [
+                             pytest.param({"extra_args": None}, id="no-additional-cli-args"),
+                             pytest.param({"extra_args": ["-nginx-debug", "-health-status=true"]},
```

Comment:
for `-nginx-debug`, is it enough just to test that the IC starts successfully?

Reply:
Yes, for the smoke test.